nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/components/monitor/LongRunningTaskMonitor.java

```diff
@@ -73,7 +75,8 @@ public void run() {
             }
         }
 
-        getLogger().info("Active threads: {}; Long running threads: {}", activeThreadCount, longRunningThreadCount);
+        final long nanos = System.nanoTime() - start;
+        getLogger().info("Active threads: {}; Long running threads: {}; time to check: {} nanos", activeThreadCount, longRunningThreadCount, NumberFormat.getInstance().format(nanos));
```

Comment:
Unfortunately, changing this logging statement broke the existing unit test, so some adjustment will be necessary.

Reply:
D'oh, thanks!